lib/setup.js

```diff
@@ -633,6 +633,26 @@ function processCommand(command, args, params, callback) {
 
                     setup.setup((isFirst, isRedis) => {
                         if (isFirst) {
+                            const initialInstances = ['admin', 'discovery', 'info'];
+
+                            // @ts-ignore
```

Comment:
Remove this and change line 4 in tsconfig.json to `"target": "es2015"`

Reply:
ok